t/ui/16-activity-view.t

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env perl
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+use Test::Most;
+
+use Test::Mojo;
+use Test::Warnings ':report_warnings';
+use Test::More;
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use OpenQA::Test::TimeLimit '20';
+use OpenQA::Test::Case;
+use OpenQA::Client;
+
+use OpenQA::SeleniumTest;
+
+OpenQA::Test::Case->new->init_data(fixtures_glob => '01-jobs.pl');
+
+plan skip_all => $OpenQA::SeleniumTest::drivermissing unless my $driver = call_driver;
+
+my $t = Test::Mojo->new('OpenQA::WebAPI');
+# we need to talk to the phantom instance or else we're using the wrong database
+my $url = 'http://localhost:' . OpenQA::SeleniumTest::get_mojoport;
+
+subtest 'Login required' => sub {
+  TODO: {
+        $driver->title_is('FIXME: NOT openQA', 'redirected to login');
+        $driver->find_element_by_link_text('Login')->click();
+        is($driver->find_element('#user-action a')->get_text(), 'Logged in as Demo', 'logged in as demo');
+        $driver->refresh;
+    }
```

Comment:
Better not add anything that can not currently work. Care to remove the subtest? Then you can merge the PR unless you change more

Reply:
I'll replace it with a test accessing the menu, that's easy and still useful